<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вход/Регистрация</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.153682, y1:0.199, x2:1, y2:1, stop:0.0227273 rgba(67, 128, 80, 255), stop:1 rgba(159, 137, 137, 255));</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(62, 255, 8); font-size: 20px;
background-color: rgb(84, 141, 111);</string>
   </property>
   <property name="text">
    <string>РЕГИСТРАЦИЯ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 33, 92); font-size: 15px;
background-color: rgb(255, 219, 87);</string>
   </property>
   <property name="text">
    <string>Логин</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 33, 92); font-size: 15px;
background-color: rgb(255, 155, 157);</string>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 33, 92); font-size: 15px;
background-color: rgb(196, 255, 171);</string>
   </property>
   <property name="text">
    <string>Подтвердите пароль</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirm_password">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="show_hide_pass_btn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>220</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show_hide_pass_btn_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registration_btn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Зарегестрироваться</string>
   </property>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0); font-size: 15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>391</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(84, 141, 111);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="login">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ВХОД</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>label</zorder>
  <zorder>label_4</zorder>
  <zorder>login</zorder>
  <zorder>username</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>password</zorder>
  <zorder>confirm_password</zorder>
  <zorder>show_hide_pass_btn</zorder>
  <zorder>show_hide_pass_btn_2</zorder>
  <zorder>status</zorder>
  <zorder>registration_btn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
